### DISCUSSION TITLE
spyder-modelx will no longer support Spyder 3

### DISCUSSION THREAD

**Discussion by fumitoh** [fumitoh/modelx#61] [Show and tell]
Starting version 0.12.0, the next release of spyder-modelx, Spyder 3 will not be supported. For those who use spyder-modelx with Spyder , it is recommended to upgade your Spyder to a supported version (Spyder 4 or 5).

**Comment by jeschenx** [comment]
Not sure if it's related... I get an error setting up. I tired sending a crash report but encountered an error there as well... I am using Spyder 5.1.5 via Anaconda. It can't seem to connect to kernel and it crashes every time I launch a new MxConsole. Screenshot attached. What am I doing wrong?
<img width="507" alt="MXConcole crashes Spyder " src="https://user-images.githubusercontent.com/106347404/181996927-5e86bcbb-3eeb-4cc3-831c-974bc28e2451.png">

**Reply by fumitoh** [reply]
Moving this to Issue https://github.com/fumitoh/modelx/issues/67
